africanlii/views/documents.py

```diff
@@ -26,6 +26,11 @@ class DocumentDetailViewResolver(View):
     """Resolver view that returns detail views for documents based on their doc_type."""
 
     def dispatch(self, request, *args, **kwargs):
+        expression_frbr_uri = kwargs.get("expression_frbr_uri")
```

Comment:
You can make the `expression_frbr_uri` param (or actually `frbr_uri`) an actual parameter to the method, rather than having to use `kwargs`. We know it's always going to be there.